<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_definePlot</class>
 <widget class="QDialog" name="Dialog_definePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>576</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>576</width>
    <height>411</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plot definition</string>
  </property>
  <widget class="QWidget" name="widget_right" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>291</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>291</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="0">
     <widget class="QSlider" name="slider_thickRight">
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Grid thickness</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="slider_ticksRight">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="combo_rightScaleStyle"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_rightScaleColor">
      <property name="text">
       <string>Set grid color</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ticks</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTreeWidget" name="list_rightData">
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Grid style</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_rightGridColor">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>list_rightData</zorder>
   <zorder>label_5</zorder>
   <zorder>btn_rightScaleColor</zorder>
   <zorder>frame_rightGridColor</zorder>
   <zorder>combo_rightScaleStyle</zorder>
   <zorder>slider_thickRight</zorder>
   <zorder>slider_ticksRight</zorder>
   <zorder>label_7</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>291</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>291</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Gird thickness</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="slider_thickLeft">
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_leftScaleColor">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Set grid color</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="combo_leftScaleStyle"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ticks</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="slider_ticksLeft">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTreeWidget" name="list_leftData">
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Grid style</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_leftGridColor">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cb_enableLeftScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable left scale</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_enableRightScale">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable right scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setBackground">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set background color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot name:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_background">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
